reconcile/cli.py

```diff
@@ -219,6 +220,14 @@ def init_log_level(log_level):
     logging.basicConfig(format=format,
                         datefmt='%Y-%m-%d %H:%M:%S',
                         level=level)
+    if 'syslog' in config.get_config():
```

Comment:
Check if Syslog server is configured

Reply:
all integrations get the same config file. that means that all integrations will attempt to send logs to syslog.
unless i'm missing something?

Reply:
Yes. Syslog goes through UDP connect. So there are no errors or warnings for no Syslog server. Maybe we should add a new config flag to make Syslog optional.